src/routes/docs/+page.server.ts

```diff
@@ -0,0 +1,23 @@
+import { toHtml } from 'hast-util-to-html';
+import { fromMarkdown } from 'mdast-util-from-markdown';
+import { toHast } from 'mdast-util-to-hast';
+import { visit } from 'unist-util-visit';
+import type { PageServerLoad } from './$types';
+
+export const load: PageServerLoad = async () => {
+  const rawChangelog = Object.values(import.meta.glob('/CHANGELOG.md', { eager: true, as: 'raw' }));
+
+  const changes: string[] = [];
+
+  const mdast = fromMarkdown(rawChangelog[0]);
+
+  visit(mdast, 'listItem', (node) => {
+    node.type = 'paragraph';
+    // @ts-ignore it's following the official docs https://github.com/syntax-tree/mdast-util-to-hast#use
+    changes.push(toHtml(toHast(node)));
```

Comment:
This might be refactorable to use the `compile-function` from `mdsvex`. I'm not sure about this though, because we want to extract data from the ast, and not modify it and return a renderable html file

Reply:
In the vague case, where this is possible we can remove all these dependencies again 🤔

Reply:
I played around with mdsvex, it seems you can directly render the html from the CHANGELOG.md file.

Something like this:
```
  const rawChangelog = Object.values(import.meta.glob('/CHANGELOG.md', { eager: true, as: 'raw' }));

  const data = await compile(rawChangelog[0]);
  const changes = data?.code ?? '<p>No change to show <p>';
```

Though it means , changing the timeline.svelte to something like this:
```
<script lang="ts">
  export let changes: string;
</script>

<div class="changelogs">
  {@html changes}
</div>

<style>
  .changelogs :global(h2) {
    color: gray;
  }
</style>

```

Reply:
Yes, that's true, I didn't want to render the entire page though. I only wanted to extract some elements of the markdown ast and render them as html, not the entire page. The compile function might still do the trick here, I didn't test it just yet though

Reply:
Ah, in that case It seems there is not an easy way to do it with compile. The compile function does have a option that uses the remark or rehype plugins. But, since I am not familiar with this I'm not sure if there are any that would help in extracting the desired elements. :(